src/integration/utils/customResourceLoader.js

```diff
@@ -0,0 +1,16 @@
+const jsdom = require('jsdom');
+
+class CustomResourceLoader extends jsdom.ResourceLoader {
+  fetch(url, options) {
+    if (
+      url ===
+      'https://mybbc-analytics.files.bbci.co.uk/reverb-client-js/reverb-3.9.2.js'
```

Comment:
Does this url need to be dynamic to account for future versions of the Reverb dependency?

Reply:
Good shout.
I'll move this to a config file to allow for a more appropriate
approach to updating the version of Reverb if/when we need to.

Reply:
This is now set dynamically from the env config as seen [here](https://github.com/bbc/simorgh/pull/12189/files#diff-e592b3765dc504d327416fd8637fb4f8b786ff1dd33b9269b44f5b418759bc9fR55).
For the tests, we're setting the URL from the [.env](https://github.com/bbc/simorgh/pull/12189/files#diff-76ae1971d88db7cda0f0d6c98a7bb928af7533e72fee96b64f49fd59d07732acR6) file.